### DISCUSSION TITLE
Default formatter doesn't return context by default.

### DISCUSSION THREAD

**Discussion by todd-a-jacobs** [reidmorrison/semantic_logger#267] [Ideas]
### Environment

- Any Ruby
- semantic_logger 4.10.0

### Expected Behavior

The default formatter and its descendants should support context by default.

### Actual Behavior

There is no element for context in the return value from the default formatter. It has to be added as a separate element in a custom formatter, in the form of `log.context` because it's not being passed in by default.

### Pull Request

I'm unsure if this is intentional or not. It's easy enough to add `log.formatter` to the returned array of elements from the default formatter, but there may be a better way to handle it, or a good reason it isn't available by default.

**Comment by reidmorrison** [comment]
`context` is specific to each appender and can hold any Ruby object inside the Hash, so rendering the context by default would be risky.
